src/common/res/features/days-of-buffering/main.js

```diff
@@ -0,0 +1,64 @@
+// DoB means Days of Buffering
+
+function ynabEnhancedDoB() {
+    var YNABheader = document.getElementsByClassName("budget-header-flexbox")[0];
+    var elementForAoM = document.getElementsByClassName("budget-header-days")[0];
+    var elementForDoB = elementForAoM.cloneNode(true);
+
+    var DoB = ynabEnhancedDoBCalculate();
+    elementForDoB.children[0].innerText = DoB + " day" + (DoB == 1 ? "" : "s");
+    elementForDoB.children[0].className = elementForDoB.children[0].className + " days-of-budgeting";
+    elementForDoB.children[1].innerText = "Days of Buffering";
+    elementForDoB.children[1].title = "Don't like AoM? Here you are this new shiny metric! Shows how many days you can live without income on your budget total considering average daily outflow.";    
+
+    YNABheader.appendChild(elementForDoB);
+}
+
+function onlyUnique(value, index, self) { 
+    return self.indexOf(value) === index;
+}
+
+function ynabEnhancedCheckTransactionTypes(transactions) {
+    // There must be exactly 3 types that are described.
+    var uniqueTransactionTypes = Array.from(transactions, (el) => el.displayItemType).filter(onlyUnique);
+    var handeledTransactionTypes = ["subTransaction", "transaction", "scheduledTransaction"];
+    var allTypesHandeled = (uniqueTransactionTypes.length == handeledTransactionTypes.length) && 
+        uniqueTransactionTypes.every(function(element, index) {
+            return element === handeledTransactionTypes[index]; 
+        }); 
+    if (!allTypesHandeled) {
+        throw "Found unhandeled transaction type. " + uniqueTransactionTypes;
```

Comment:
This threw "Uncaught Found unhandeled transaction type. subTransaction,transaction" on my machine.